lib/authorizer/digest.js

```diff
@@ -1,4 +1,6 @@
 var crypto = require('crypto-js'),
+
+    NON_INTERACTIVE = 'nonInteractive',
```

Comment:
IMO, now that this flag is not generic but specific to individual auths, it should not be called `nonInteractive`.

Reply:
@kamalaknn what should it be called?

Reply:
These should be like @kamalaknn says - specific to each auth
digest may have some terminology in the RFC

Reply:
The RFC also uses the retry in its wording.

```
The client is expected to retry the request, passing an Authorization header line
```

It should be `disableRetry{something}`.

Reply:
@kamalaknn Same is true for NTLM also
https://msdn.microsoft.com/en-us/library/cc237544.aspx
![image](https://user-images.githubusercontent.com/5207331/30899442-8815518e-a37d-11e7-864a-5fa2c0345c1b.png)